src/traces/sunburst/attributes.js

```diff
@@ -129,6 +130,9 @@ module.exports = {
         editType: 'plot',
         flags: ['label', 'text', 'value']
     }),
+    texttemplate: texttemplateAttrs({editType: 'plot'}, {
+        keys: ['label', 'text', 'value', 'color']
```

Comment:
Similar here, does anyone else think it's odd to have `%{text}` in a `texttemplate`?

Reply:
> **but** one could also do
> 
> ```js
> {
>   customdata: ['a', 'b', 'c'],
>   texttemplate: 'My text: %{customdata}'
> }
> ```

Actually, this isn't working in the current implementation! It needs to be fixed!

Reply:
Fixed in e0bde45fe84d78d07f4d02882cd76a62f0345fa0